staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -1199,6 +1203,9 @@ func Test_growSlice(t *testing.T) {
 			if tt.cap != tt.args.v.Cap() {
 				t.Errorf("Unexpected capacity: got=%d want=%d", tt.args.v.Cap(), tt.cap)
 			}
+			if tt.len != tt.args.v.Len() {
```

Comment:
Please also compare the elements to verify they didn't change (might require putting some values in).

Reply:
Done.